portfolio/src/main/webapp/index.html

```diff
@@ -35,6 +35,76 @@ <h2>About me.</h2>
           in that language, I am learning data structures and I find myself more comfortable using C++ 
           than other languages, also I have participated in competitive programming contest and I use C++ too.</p>
       </article>
+      <article>
+        <h2>Projects I've worked on.</h2>
+        <p>These are some projects I've worked on including
+          the list of technologies I used on every project.</p>
+        <ol>
+          <li>Backend for school exchange programs
+            <ul>
+              <li>NodeJS</li>
+              <li>LoopBack Framework</li>
+            </ul>
+          </li>
+          <li>Six degree of freedom robot arm simulation (inverse kinematics)
+            <ul>
+              <li>Matlab</li>
+              <li>Numerical Methods</li>
+            </ul>
+          </li>
+          <li>Backend for school exchange programs.
+            <ul>
+              <li>NodeJS</li>
+              <li>LoopBack Framework</li>
+            </ul>
+          </li>
```

Comment:
This seems to have been listed twice. Did you intend to add a duplicate?

Reply:
Oh, I forgot to change it. edited, thanks.